packages/x-components/tests/e2e/next-queries-preview.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Next queries preview component
```

Comment:
Should we add a case when you click the query of the next query? Because right now the query can be clicked and set the search query.